resolwe/flow/serializers.py

```diff
@@ -1,318 +0,0 @@
-""".. Ignore pydocstyle D400.
```

Comment:
Change commit message to: `Split flow's serializers into module`

Reply:
From Python docs: "[A module is a file containing Python definitions and statements](https://docs.python.org/3/tutorial/modules.html)". If so, it would be better to rename commit message to "Split flow's serializers into module**s**"(plural). Would you agree?

Reply:
You are right, or you can use `package` instead.